### PR TITLE
Drop support for Node.js 10

### PR DESCRIPTION
**Pull request by fisker**
## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: scripts/build/build.mjs [hunk 1]
```diff
@@ -126,7 +126,7 @@ function handleError(error) {
 async function preparePackage() {
   const packageJson = await readJson(path.join(PROJECT_ROOT, "package.json"));
   packageJson.bin = "./bin-prettier.js";
-  packageJson.engines.node = ">=10.13.0";
+  // packageJson.engines.node = ">=10.13.0";
```
Comment (sosukesuzuki) [inline scripts/build/build.mjs:129]: Why commented out?

Comment (fisker) [reply scripts/build/build.mjs:129]: I can remove it if you prefer.

Comment (sosukesuzuki) [reply scripts/build/build.mjs:129]: Ah sorry, I just want to know the reason.

Comment (fisker) [reply scripts/build/build.mjs:129]: Maybe we need a different node version for production in future, we can comment it back.

Comment (sosukesuzuki) [reply scripts/build/build.mjs:129]: I got it

**Review by sosukesuzuki — APPROVED** [review]

**Pull request merged**